ci-operator/config/openshift/openshift-tests-private/AGENTS.md

```diff
@@ -0,0 +1,220 @@
+
+**CPOU Upgrade:**
+`openshift-openshift-tests-private-release-4.20__amd64-nightly-4.20-cpou-upgrade-from-4.18.yaml`
+
+**Rollback:**
+`openshift-openshift-tests-private-release-4.20__amd64-rollback-nightly.yaml`
+
+## Upgrade Job Maintenance Strategies
+
+When maintaining upgrade test jobs, follow these strategic guidelines to ensure comprehensive coverage while avoiding redundancy:
+
+### Common Requirements for all Jobs
+1. **Disconnected profile distribution**
+   - Run most disconnected profiles on **non-multi** architecture
+   - Don't use disconnected profiles for chain upgrade jobs
+
+2. **Multi-architecture preference**
+   - Use multi-architecture for **at least 65%** of profiles
+
+3. **ARM64 preference**
+    - Try to use ARM64 architecture when possible
+
+4. **TechPreview (TP) features**
+    - TP jobs are **only supported for z-stream upgrades**
+
+5. **Architecture-sensitive Features**
+    - For those profiles which include features that are affected by architecture, we copy them on all architectures
+    - The architecture sensitive features include storage (aws efs, gcp filestore-csi, bm LSO), mco, and cluster infrastructure
+
+### Requirements for Chain Upgrade Jobs
+
+1. **Applicable Versions**
+   - Only latest version has Chain upgrade jobs
+
+2. **Cross-platform, cross-feature, cross-architecture coverage**
+   - Chain upgrade paths must cover all platforms: AWS, Azure, Baremetal, GCP, IBMCloud, Nutanix, vSphere
+   - Chain upgrade jobs must cover all key features: sdn (4.15 and earlier; 4.14 default is OVN), sno, ovn, ipsec, ipv4, ipv6, dual-stack, fips, proxy, capability, sts, mixarch
+   - Chain upgrade jobs must cover all CPU architectures: amd64, arm64, multiarch
+
+3. **EOL (End of Life) version handling**
+   - Only keep one more release for EOL versions
+   - Reference: [OpenShift Life Cycle Dates](https://access.redhat.com/support/policy/updates/openshift)
+
+4. **Stable-to-stable upgrade paths**
+   - Choose only ONE profile for stable-to-stable upgrade paths in chain upgrades
+   - The profile should copy from stable-to-nightly
+
+5. **Customer usage-driven coverage**
+   - No need to cover versions not actively used by customers
+   - Reference: [Current version distribution dashboard](https://superset.datahub.redhat.com/superset/dashboard/ccx-upgrades-overview/)
+   - Discussion: [Slack thread](https://redhat-internal.slack.com/archives/CF8SMALS1/p1731671996685359)
+   - Reference: [OCP sequential upgrade test matrix proposal](https://docs.google.com/document/d/1KfYnKCSu3Iv3DA9oFWOTqzwfLS7vRbmcCWH-q-T7MvY/edit?tab=t.0#heading=h.jsje0qn5w5o3)
+
+### Update Upgrade Files
+
+1. **ansible, cli, tools images**
+   - For versions **earlier than 4.6**: Use **target version** for ansible, cli, tools
+   - For versions **4.6 and newer**: Use **initial version** for ansible, cli, tools
+
+2. **upi-installer, openstack-installer images**
+   - **Always use initial version** regardless of OCP version
```

Comment:
Not clear what this means, can we reword, or give an example?

Reply:
same as above in base_images section,  which version needs to be used for.  

  upi-installer:
    name: "4.18"
    namespace: ocp
    tag: upi-installer